### ISSUE TITLE
"Missing sitekey" error with Angular 11.2.14

### ISSUE THREAD

**Issue by m-thalmann** [DSergiu/vanilla-hcaptcha#8]
Hi,
when using the web-component as described in the documentation, I get the following error in the console:
`[hCaptcha] Missing sitekey - https://hcaptcha.com/docs/configuration#jsapi`

How I included it:
```html
<h-captcha
            [attr.site-key]="siteKey"
            (verified)="verified($event)"
            *ngIf="showHCaptcha"
        ></h-captcha>
```

If I am using it like this it works:
```html
<h-captcha
            site-key="<my-site-key>"
            (verified)="verified($event)"
            *ngIf="showHCaptcha"
        ></h-captcha>
```

Is there a way to fix this? Thanks.

**Comment by m-thalmann** [comment]
It seems like there is a delay, when binding the value. By wrapping a timeout (with 0ms) around the contents of the `connectedCallback()`-function the bug is solved:

```js
connectedCallback() {
    setTimeout(() => {
        this.config = {
            sitekey: this.getAttribute('sitekey') || this.getAttribute('site-key'),
            theme: this.getAttribute('theme') || this.getAttribute('dark') !== null ? 'dark' : undefined,
            size: this.getAttribute('size'),
            tabindex: this.getAttribute('tabindex'),
            language: this.getAttribute('language'),
            reCaptchaCompat: this.getAttribute('reCaptchaCompat') === "false",
            challengeContainer: this.getAttribute('challengeContainer'),
            rqdata: this.getAttribute('rqdata'),
            sentry: this.getAttribute('sentry') === "true",
            apiEndpoint: this.getAttribute('apiendpoint') || 'https://hcaptcha.com/1/api.js',
            endpoint: this.getAttribute('endpoint'),
            reportapi: this.getAttribute('reportapi'),
            assethost: this.getAttribute('assethost'),
            imghost: this.getAttribute('imghost'),
        };
        loadApiEndpointIfNotAlready(this.config).then(this._apiLoaded.bind(this)).catch(this.onError.bind(this));
    }, 0);
}
```

**Comment by DSergiu** [comment]
The issue seems to be with angular which would render the component with no attributes at first.

This should be fixed in the latest `1.0.0-alpha.3`. Please let me know if you still have the issue with the latest version.
